app/controllers/works_controller.rb

```diff
@@ -46,7 +46,9 @@ def work_params
                                  'created(1i)', 'created(2i)', 'created(3i)',
                                  'created_range(1i)', 'created_range(2i)', 'created_range(3i)',
                                  'created_range(4i)', 'created_range(5i)', 'created_range(6i)',
-                                 :created_edtf, :abstract, :citation, :access, :license, :agree_to_terms,
+                                 :created_edtf, :abstract, :citation, :access, :license,
+                                 :release, 'embargo_date(1i)', 'embargo_date(2i)', 'embargo_date(3i)',
```

Comment:
Sorry for the ignorance, can you point me towards any docs describing this syntax for attributes: `(1i)` etc...

Reply:
https://stackoverflow.com/questions/51959940/rails-date-select-parameter-permit

Reply:
I see, I knew about the permitted attributes stuff, but hadn't seen these date fields split into year/month/day like this.